Makefile

```diff
@@ -35,11 +35,11 @@ lint: node_modules/.uptodate
 
 .PHONY: docs
 docs: python
-	tox -e py36-docs
+	tox -e docs
```

Comment:
We shouldn't need to specify the version here as we now use `tox-pyenv` will now find the version specified in `.python-version` when we don't specify.